test/outline/test_outline_html.py

```diff
@@ -1,13 +1,13 @@
 from pathlib import Path
 
-from fpdf import FPDF, HTMLMixin, HTML2FPDF
+from fpdf import FPDF, HTML2FPDF
 from test.conftest import assert_pdf_equal
 
 
 HERE = Path(__file__).resolve().parent
 
 
-class MyFPDF(FPDF, HTMLMixin):
```

Comment:
This class is useless then, please just replace `MyFPDF` usages by `FPDF`

Reply:
It's fixed at 11d7d248b5ddf08d9e9af5510674d85e1ba0aea3